safe_transaction_service/history/tests/test_migrations.py

```diff
@@ -105,3 +130,89 @@ def test_migration_backward_0068(self):
         # Empty object should be None
         hash = Web3.keccak(text=f"multisig-tx-{origins[2]}").hex()
         self.assertEqual(MultisigTransactionOld.objects.get(pk=hash).origin, None)
+
+    def test_migration_forward_0069(self):
+        old_state = self.migrator.apply_initial_migration(
+            ("history", "0068_alter_multisigtransaction_origin")
+        )
+
+        EthereumBlock = old_state.apps.get_model("history", "EthereumBlock")
+        EthereumTx = old_state.apps.get_model("history", "EthereumTx")
+        ethereum_tx = self.build_ethereum_tx(EthereumBlock, EthereumTx)
+        SafeContract = old_state.apps.get_model("history", "SafeContract")
+        SafeContract.objects.create(
+            address=Account.create().address,
+            erc20_block_number=8,
+            ethereum_tx=ethereum_tx,
+        )
+        SafeContract.objects.create(
+            address=Account.create().address,
+            erc20_block_number=4,
+            ethereum_tx=ethereum_tx,
+        )
+        SafeContract.objects.create(
+            address=Account.create().address,
+            erc20_block_number=15,
+            ethereum_tx=ethereum_tx,
+        )
+        new_state = self.migrator.apply_tested_migration(
+            ("history", "0069_indexingstatus_and_more"),
```

Comment:
The `_and_more` in the migration name is because remove `erc20_block_number` ?

Reply:
Yes, it was an automatic name from Django, I didn't rename it